java-sdk-example/src/test/java/AdvancedCreateSandboxTest.java

```diff
@@ -0,0 +1,191 @@
+import org.testng.annotations.AfterSuite;
+import org.testng.annotations.BeforeSuite;
+import org.testng.annotations.Test;
+
+import java.util.List;
+
+import static io.restassured.RestAssured.given;
+import static org.hamcrest.Matchers.*;
+
+public class AdvancedCreateSandboxTest {
+
+    public static final String ORG_NAME = "signadot";
+    public static final String HOTROD = "hotrod";
+    public static final String SIGNADOT_API_KEY = System.getenv("SIGNADOT_API_KEY");
+    private static RequestSpecification requestSpec;
+
+    ApiClient apiClient;
+    SandboxesApi sandboxesApi;
+    CreateSandboxResponse response;
+    String sandboxID;
+
+    @BeforeSuite
+    public void createSandbox() throws ApiException, InterruptedException {
+        apiClient = new ApiClient();
+        apiClient.setApiKey(SIGNADOT_API_KEY);
+        sandboxesApi = new SandboxesApi(apiClient);
+
+        String sandboxName = String.format("test-ws-%s", RandomStringUtils.randomAlphanumeric(5));
+
+        SandboxFork frontendFork = new SandboxFork()
+            .forkOf(new ForkOf().kind("Deployment").namespace(HOTROD).name("frontend"))
+            .customizations(new SandboxCustomizations()
+                .addImagesItem(new Image().image("signadot/hotrod-frontend:latest"))
+                .addEnvItem(new EnvOp().name("ROUTE_ADDR").valueFrom(
+                    new EnvValueFrom().fork(
+                        new EnvValueFromFork().forkOf(
+                            new ForkOf()
+                                .kind("Deployment")
+                                .namespace("hotrod")
+                                .name("route")
+                        ).expression("{{ .Service.Host }}:{{ .Service.Port }}")
+                    )
+                ))
+            ).addEndpointsItem(new ForkEndpoint().name("hotrod-frontend").port(8080).protocol("http"));
+
+        SandboxFork routeFork = new SandboxFork()
+                .forkOf(new ForkOf().kind("Deployment").namespace(HOTROD).name("route"))
+                .customizations(new SandboxCustomizations()
+                        .addEnvItem(new EnvOp().name("abc").value("def").operation("upsert"))
+                        .addImagesItem(new Image().image("signadot/hotrod:0ed0bdadaa3af1e4f1e6f3bb6b7d19504aa9b1bd"))
+                        .patch(new CustomPatch().type("strategic").value("""
+                            spec:
+                              template:
+                                spec:
+                                  containers:
+                                  - name: hotrod
+                                    env:
+                                    - name: TEST
+                                      value: foo
+                            """)))
+                .addEndpointsItem(new ForkEndpoint().name("hotrod-route").port(8083).protocol("http"));
+
+        CreateSandboxRequest request = new CreateSandboxRequest()
+                .cluster("demo")
```

Comment:
The cluster name also needs to be parameterized and passed as env var (as in other tests)